src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -250,12 +255,37 @@ export const thorchainSaversStakingOpportunitiesUserDataResolver = async ({
       const rewardsAmountsCryptoBaseUnit: [string] = [
         stakedAmountCryptoBaseUnitIncludeRewards.minus(stakedAmountCryptoBaseUnit).toFixed(),
       ]
+      let maturity: number | undefined = undefined
+
+      if (stakingOpportunityId === thorchainAssetId && accountPosition.last_add_height) {
+      }
+
+      const maybeMaturity = await (async () => {
+        if (stakingOpportunityId !== thorchainAssetId) return {}
+        // note undefined over missing property here, i.e this is a runepool position and explicitly undefined
+        if (!accountPosition.last_add_height) return
```

Comment:
```suggestion
        if (!accountPosition.last_add_height) return {}
```

Reply:
Fixed in [60e41ac](https://github.com/shapeshift/web/pull/7472/commits/60e41aceb721592767aac449c8b1671168120425)